src/Sarif.UnitTests/Readers/SampleModel/LogModel.cs

```diff
@@ -114,7 +114,7 @@ protected override JsonContract CreateContract(Type objectType)
 
     internal class LogModelSampleBuilder
     {
-        public const string SampleLogPath = "CodeCrawler.log.json";
+        public const string SampleLogPath = @"e:\repros\CodeCrawler.log.json";
```

Comment:
>repros [](start = 49, length = 6)

Is this going to be another one of those cases where the test won't work on a machine that doesn't have your drives and directory layout?